shop/src/pages/admin/settings/appearance/Edit.js

```diff
@@ -295,7 +289,10 @@ const AppearanceSettings = () => {
                   }
                 ]
               }}
-              onChange={(e) => setAboutText(e.editor.getData())}
+              onChange={(e) => {
+                setAboutText(e.editor.getData())
+                setState({ hasChanges: true })
```

Comment:
Known issue: 
As noted in Line 66, "CK editor takes a few seconds to load". 
As a consequence, the app will enable the Cancel and Update buttons on page load if a merchant has already populated the 'About your Store' field under Settings > Appearance.